src/main/target/common_defaults_post.h

```diff
@@ -428,6 +428,12 @@
 #ifndef BARO_I2C_INSTANCE
 #define BARO_I2C_INSTANCE       I2C_DEVICE
 #endif
+#ifndef BARO_EOC_PIN
```

Comment:
Again, this will never be used as `BARO_EOC_PIN` is already defined in `src/main/target/common_post.h`.